docs/content/guides/rows/row-parent-child.md

```diff
@@ -378,25 +376,21 @@ ReactDOM.render(<ExampleComponent />, document.getElementById('example1'));
 
 :::
 
-In the example above, we’ve created a data object consisting of 2016’s Grammy nominees of the “Rock”
-genre. Each _0-level_ entry declares a category, while their children declare nominees - assigned
-under the `__children` properties.
+In the example above, we’ve created a data object consisting of 2016’s Grammy nominees of the “Rock” genre. Each _0-level_ entry declares a category, while
+their children declare nominees - assigned under the `__children` properties.
 
-Note that the first 0-level object in the array needs to have all columns defined to display the
-table properly. They can be declared as `null` or an empty string `''`, but they need to be defined.
-This is optional for the other objects.
+Note that the first 0-level object in the array needs to have all columns defined to display the table properly. They can be declared as `null` or an empty
+string `''`, but they need to be defined. This is optional for the other objects.
 
 ## User interface
 
-The _Nested Rows_ plugin's user interface is placed in the row headers and the Handsontable’s
-context menu.
+The _Nested Rows_ plugin's user interface is placed in the row headers and the Handsontable’s context menu.
 
 ### Row headers
 
 Each _parent_ row header contains a `+`/`-` button. It is used to collapse or expand its child rows.
 
-The child row headers have a bigger indentation, to enable the user to clearly recognize the child
-and parent elements.
+The child row headers have a bigger indentation, to enable the user to clearly recognize the child and parent elements.
```

Comment:
Hello, here is my proposition:

"The child row headers have larger indentations in order to enable the clear recognition of the child and parent elements for the user."